.vscode/launch.json

```diff
@@ -27,6 +27,19 @@
       ],
       "outFiles": ["${workspaceFolder}/out/test/**/*.js"],
       "preLaunchTask": "npm: test-compile"
+    },
```

Comment:
The Live Share tests are slower than `Peacock`'s ones since they imply sharing a workspace with Live Share. Because of this fact and since I wanted to keep the integration separated from eh main code base, I've created this separate launch definition.

Reply:
Thanks. I think I'd want all of the tests to run when this is in CI. perhaps the main tests can kick off the live-share ones, too?